src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionSubmissionEditPageActionTest.java

```diff
@@ -31,6 +31,7 @@ public static void classSetUp() throws Exception {
 
     @Test
     public void testExecuteAndPostProcess() throws Exception {
+        @SuppressWarnings("unused")
         AccountAttributes instructorAccount = dataBundle.accounts.get("instructor1OfCourse1");
```

Comment:
It is not used until later on when instructorAccount is changed to something else, only the initial instantiation is not used if I'm not wrong